manifests/04-rbac-rolebinding.yaml

```diff
@@ -37,3 +37,30 @@ subjects:
 - kind: ServiceAccount
   name: console-operator
   namespace: openshift-console-operator
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config-managed
+roleRef:
+  kind: Role
+  name: console-operator
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: User
+    name: system:serviceaccount:openshift-console-operator:console-operator
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-console
```

Comment:
Don't we already have this role binding above?

Reply:
I was using a different name prior to the last updates. Now this is a repeat and has been removed.